AK/StringView.cpp

```diff
@@ -40,16 +40,16 @@ StringView::StringView(ByteBuffer const& buffer)
 {
 }
 
-Vector<StringView> StringView::split_view(char const separator, bool keep_empty) const
+Vector<StringView> StringView::split_view(char const separator, SplitBehavior action_on_split) const
```

Comment:
Nit-pick, but it bothers me a little that this is called `action_on_split` in a few places, instead of `split_behavior`.

Reply:
My sed command didn't work very well. :confused: